hyperjumptech/monika
# Fix TLS check showing empty message

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. TLS check was not showing a message

## How did you implement / how did you fix it  
1.  Changed empty string to error.message

## How to test  
1. Run an instance of monika (not test scripts) with an expiring/expired domain check.
2. Look at the `Message:`
3. With the fix in place, you will get a useful message.
